great_expectations_cloud/agent/message_service/asyncio_rabbit_mq_client.py

```diff
@@ -16,6 +17,9 @@
     from pika.channel import Channel
     from pika.spec import Basic, BasicProperties
 
+LOGGER: Final[logging.Logger] = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
```

Comment:
Not for this PR, but at some point we'll likely want to control these with an environment variable so that users will be able to change the logging level for their deployments